docs/en/running-node/open-node-tor-onion.md

```diff
@@ -256,7 +234,7 @@ Proto Recv-Q Send-Q Local Address           Foreign Address         State
 ...
 tcp        0      0 0.0.0.0:18080           0.0.0.0:*               LISTEN      259255/monerod
 tcp        0      0 0.0.0.0:18081           0.0.0.0:*               LISTEN      259255/monerod
-tcp        0      0 127.0.0.1:18083         0.0.0.0:*               LISTEN      259255/monerod
+tcp        0      0 127.0.0.1:18084         0.0.0.0:*               LISTEN      259255/monerod
```

Comment:
what's wrong with the default port of 18083 again?  slot of zmq operators have moved to away from the default port to 18084 https://www.reddit.com/r/Monero/comments/vga773/psa_to_public_node_operators_that_have_enabled/

Reply:
Default ports:
18080 p2p
18081 rpc
18082 zmq-rpc

User added:
18083 zmq-pub
18084 tor p2p
18085 i2p p2p
18089 restricted rpc

Keeping zmq's together and torp2p+i2pp2p together is cleaner than going zmq > tor > zmq > i2p